src/style.css

```diff
@@ -0,0 +1,98 @@
+body {
+  --snake-height: 300px;
+  margin: 0;
+}
+
+.snake {
+  max-width: 1199px;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.snake-item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  min-width: var(--snake-height);
+  height: var(--snake-height);
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
move to tag selector `html`

Reply:
i can't move it to html selector because style property is banned